fibonacci.asm

```diff
@@ -17,15 +17,25 @@
 	.data
 
 fibs:	.word   0 : 9         # create an array variable named "fibs" of 9 word-length elements (4 bytes each)
-size:	.word  9              # create a single integer variable named "size" that indicates the length of the array
+#size:	.word  9              # create a single integer variable named "size" that indicates the length of the array
+ask_input:
+	.asciiz "\Enter size of the array\n"
 
 ###############################################
 # .text segment. Assembly instructions go here.
 ###############################################
 	.text
+              la   $a0, ask_input
+              li   $v0, 4
+              syscall
+              
+              li   $v0, 5
+              syscall
+              
+              move $s5, $v0
```

Comment:
-5 Accepting input greater than declared array size is dangerous because your writing to memory thats out of bounds.